AMQPPerfHarness/src/com/ibm/uk/hursley/perfharness/amqp/FloodPublisher.properties

```diff
@@ -7,10 +7,13 @@
 # http://opensource.org/licenses/MIT
 ########################################################## {COPYRIGHT-END} ###
 
-com.ibm.uk.hursley.perfharness.amqp.Requestor.desc =\
-Simulate a requestor harness where messages are replaced on the request topic \n\
-and messages are are taken off a reply topic
+com.ibm.uk.hursley.perfharness.amqp.FloodPublisher.desc =\
+Generates message to be published to a topic.
 
 tpt.dflt = false
 tpt.desc = Topic per thread
 tpt.type = java.lang.Boolean
+
+mr.dflt = 
```

Comment:
There is already an -rt flag available for the WorkerThread that FloodPublisher implements which provides rate control. Does this already provide the functionality you are looking to provide here?